### PR TITLE
Fix bundles

### PR DESCRIPTION
**Pull request by InoMurko**
swaps swagger-combine with swagger-cli

## Overview
swaps swagger-combine with swagger-cli. I forgot to incl

## Changes

- Regenerate api descriptions to avoid `refs`, because generators are not able to digest them
- Rename `Watcher's` to `Watchers` because generated code doesn't digest `'`

## Testing

/

### CODE REVIEW

File: apps/omg_watcher_rpc/priv/swagger/informational_api_specs/swagger.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   version: '1.0.0'
-  title: Watcher's Informational API
+  title: Watchers Informational API
```
Comment (achiurizo) [inline apps/omg_watcher_rpc/priv/swagger/informational_api_specs/swagger.yaml]:
```suggestion
  title: Watcher Informational API
```

Just make it simple and drop the 's'

File: apps/omg_child_chain_rpc/priv/swagger/operator_api_specs.yaml [hunk 2]
```diff
@@ -90,16 +83,31 @@ paths:
...
+        '500':
           description: Returns an internal server error
           content:
             application/json:
               schema:
                 description: The response schema for an error
                 allOf:
-                  - *ref_0
+                  - description: The response schema for a successful operation
+                    type: object
```
Comment (chrishunt) [inline apps/omg_child_chain_rpc/priv/swagger/operator_api_specs.yaml]: It looks like our objects in the generated client code have questionable names like `inline_obect1`, `inline_object2`, `inline_object3`, ...etc. I wonder if there's something we can put in the spec to name them?

Comment (InoMurko) [reply apps/omg_child_chain_rpc/priv/swagger/operator_api_specs.yaml]: some have meaningful names, some don't ... need to diff and check!

**Comment by chrishunt** [comment]
Thank you @InoMurko! Merging this so we can use in https://github.com/omisego/specs/issues/7

**Pull request merged**